src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -42,6 +42,7 @@ public sealed class TensorFlowTransformer : RowToRowTransformerBase
         private readonly string _savedModelPath;
         private readonly bool _isTemporarySavedModel;
         private readonly bool _addBatchDimensionInput;
+        private readonly IHostEnvironment _env;
```

Comment:
>_env [](start = 42, length = 4)

The base class `RowToRowTransformerBase` has this field:
```
        private protected readonly IHost Host;
```
Can it be used instead of adding a new field?

Reply:
Yes, that is good point, I will modify the code as you suggested, thanks.

---
In reply to: [362313339](https://github.com/dotnet/machinelearning/pull/4613#discussion_r362313339) [](ancestors = 362313339)